Okta.Auth.Sdk/Okta.Auth.Sdk.csproj

```diff
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
     <Version>1.1.0</Version>
   </PropertyGroup>
 
   <PropertyGroup>
     <Description>Official .NET SDK for the Okta Authentication API.</Description>
-    <DocumentationFile>bin\$(Configuration)\netstandard1.3\Okta.Auth.Sdk.xml</DocumentationFile>
+    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\Okta.Auth.Sdk.xml</DocumentationFile>
```

Comment:
Minor question: Is this handled by Visual Studio automatically, or is this a manual edit to the project file?

Reply:
This is a manual edit.